db/migrations/20240515080100_create_mat_views_tags_view.js

```diff
@@ -0,0 +1,221 @@
+// coverted from
+// https://github.com/thematters/matters-metabase/blob/main/sql/stale-tags-create-table-view.sql
```

Comment:
better be permalink  ?

Reply:
it's the same; since the `matters-metabase` repo is deprecated, on new development will be there; and should be archived